step.sh

```diff
@@ -93,7 +93,7 @@ curl $BITBUCKET_API_ENDPOINT \
   --data-binary \
       $"{
         \"state\": \"$BITBUCKET_BUILD_STATE\",
-        \"key\": \"Bitrise - Build $triggered_workflow_id\",
+        \"key\": \"Bitrise - $BITRISE_BUILD_SLUG - Build $triggered_workflow_id - #$BITRISE_BUILD_NUMBER\",
         \"name\": \"Bitrise $app_title ($triggered_workflow_id) #$build_number\",
         \"url\": \"$build_url\",
         \"description\": \"workflow: $triggered_workflow_id\"
```

Comment:
in a future PR I plan on adding support for passing "$num_passed_tests tests passed"

this means our build status appears as follows:
<img width="133" alt="image" src="https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status/assets/274318/8ed71a36-39b2-4f42-b6b7-e7911c20700b">

Reply:
https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status/issues/10